property.go

```diff
@@ -0,0 +1,93 @@
+package opslevel
+
+import "fmt"
+
+type PropertyDefinitionInput struct {
+	Name   string `json:"name"`
+	Schema JSON   `json:"schema"`
+}
+
+type PropertyDefinition struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+	Name    string   `graphql:"name"`
+	Schema  JSON     `graphql:"schema" scalar:"true"`
+}
+
+type PropertyDefinitionConnection struct {
+	Nodes      []PropertyDefinition
+	PageInfo   PageInfo
+	TotalCount int `graphql:"-"`
+}
+
+func (client *Client) CreatePropertyDefinition(input PropertyDefinitionInput) (*PropertyDefinition, error) {
+	var m struct {
+		Payload struct {
+			Definition PropertyDefinition `graphql:"definition"`
+			Errors     []OpsLevelErrors   `graphql:"errors"`
+		} `graphql:"propertyDefinitionCreate(input: $input)"`
+	}
+	v := PayloadVariables{
+		"input": input,
+	}
+	err := client.Mutate(&m, v, WithName("PropertyDefinitionCreate"))
+	return &m.Payload.Definition, HandleErrors(err, m.Payload.Errors)
+}
+
+func (client *Client) GetPropertyDefinition(input string) (*PropertyDefinition, error) {
+	var q struct {
+		Account struct {
+			Definition PropertyDefinition `graphql:"propertyDefinition(input: $input)"`
+		}
+	}
+	identifier := *NewIdentifier(input)
+	v := PayloadVariables{
+		"input": identifier,
+	}
+	err := client.Query(&q, v, WithName("PropertyDefinitionGet"))
+	if q.Account.Definition.Id == "" {
+		err = fmt.Errorf("PropertyDefinition with ID or Alias matching '%s' not found", input)
+	}
+	return &q.Account.Definition, HandleErrors(err, nil)
+}
+
+func (client *Client) ListPropertyDefinitions(variables *PayloadVariables) (PropertyDefinitionConnection, error) {
+	var q struct {
+		Account struct {
+			Definitions PropertyDefinitionConnection `graphql:"propertyDefinitions(after: $after, first: $first)"`
+		}
+	}
+	if variables == nil {
+		variables = client.InitialPageVariablesPointer()
+	}
+	if err := client.Query(&q, *variables, WithName("PropertyDefinitionList")); err != nil {
+		return PropertyDefinitionConnection{}, err
+	}
+	for q.Account.Definitions.PageInfo.HasNextPage {
+		(*variables)["after"] = q.Account.Definitions.PageInfo.End
+		resp, err := client.ListPropertyDefinitions(variables)
+		if err != nil {
+			return PropertyDefinitionConnection{}, err
+		}
+		q.Account.Definitions.Nodes = append(q.Account.Definitions.Nodes, resp.Nodes...)
+		q.Account.Definitions.PageInfo = resp.PageInfo
+		q.Account.Definitions.TotalCount += len(q.Account.Definitions.Nodes)
+	}
+	return q.Account.Definitions, nil
+}
+
+func (client *Client) DeletePropertyDefinition(input string) error {
+	var m struct {
+		Payload struct {
+			DeletedAlias string           `graphql:"deletedAlias"`
+			DeletedID    ID               `graphql:"deletedId"`
+			Errors       []OpsLevelErrors `graphql:"errors"`
+		} `graphql:"propertyDefinitionDelete(resource: $input)"`
+	}
+	identifier := *NewIdentifier(input)
+	v := PayloadVariables{
+		"input": identifier,
+	}
```

Comment:
```suggestion
        v := PayloadVariables{
            "input": *NewIdentifier(input),
        }
```
Same as above.
nit: This is more in line with how `PayloadVariables` is defined in `infra.go`, `system.go`, and a couple others